aci/resource_aci_vnsabsnode.go

```diff
@@ -567,13 +582,16 @@ func resourceAciFunctionNodeUpdate(ctx context.Context, d *schema.ResourceData,
 		d.Set("conn_provider_dn", vnsAbsFuncConn.DistinguishedName)
 	}
 
-	if d.HasChange("conn_consumer_dn") || d.HasChange("conn_provider_dn") {
+	if d.HasChange("conn_consumer_dn") || d.HasChange("conn_provider_dn") || d.HasChange("conn_copy_dn") {
 		consOld, _ := d.GetChange("conn_consumer_dn")
 		d.Set("conn_consumer_dn", consOld.(string))
 
 		provOld, _ := d.GetChange("conn_provider_dn")
 		d.Set("conn_provider_dn", provOld.(string))
-		return diag.FromErr(fmt.Errorf("conn_consumer_dn and conn_provider_dn is not user configurable"))
+
+		copyOld, _ := d.GetChange("conn_copy_dn")
+		d.Set("conn_copy_dn", copyOld.(string))
+		return diag.FromErr(fmt.Errorf("conn_consumer_dn, conn_provider_dn and conn_copy_dn are not user configurable"))
```

Comment:
Is this only on update/change? Should we add that to the error message?

Reply:
they are read-only attributes even on create, made these parameters computed and not optional.